### ISSUE TITLE
Exception handling: False positive with @throws.

### ISSUE THREAD

**Issue by cburschka-pwc** [kalessil/phpinspectionsea#679]
On the latest version of the plugin (running on PhpStorm 2017.3, PS-173.3727.138), I get the "unhandled exception" notice in every function that throws exceptions, even if `@throws` is used in the PHPDoc.

Eg:

```
  /**
   * @throws \InvalidArgumentException
   */
  public function test() {
    throw new \InvalidArgumentException();
  }
```

This code will still trigger the inspection.

**Comment by kalessil** [comment]
Thank you for reporting @cburschka-pwc, but this issue supposed to be fixed with 2.3.13.2 release. Which plugin version do you have?

**Comment by cburschka-pwc** [comment]
Oops, sorry; I thought I was using the latest version since "Check for updates" didn't do anything, but in fact the plugin had already been downloaded and was pending a restart. :)

Confirmed that the error is fixed once the update is applied.

**Comment by kalessil** [comment]
Duplicates #652

**Comment by kalessil** [comment]
@cburschka-pwc : super, thanks for checking =)
